sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/FunctionRegistry.scala

```diff
@@ -384,7 +384,9 @@ object FunctionRegistry {
     expression[Rand]("rand"),
     expression[Rand]("random", true, Some("3.0.0")),
     expression[Randn]("randn"),
+    expression[RandStr]("randstr"),
     expression[Stack]("stack"),
+    expression[Uniform]("uniform"),
```

Comment:
Should also add it into Scala/Python functions in both Spark Connect and Classic. Would be great if we can file JIRAs to make sure we don't forget

Reply:
Sounds good I created this Jira in https://issues.apache.org/jira/browse/SPARK-49552